### ISSUE TITLE
drawing bounding boxes - process of creating dataset

### ISSUE THREAD

**Issue by sophia-wright-blue** [SihamTabik/Pistol-Detection-in-Videos#5]
thank you for releasing the dataset. can you summarize the steps to annotating (labeling) the dataset, how did you go about drawing the bounding boxes? was it a completely manual process? thanks,

**Comment by SihamTabik** [comment]
Hi Sophia,
The annotation process for the detection task is completly manual. You can find some more details in our paper.
Best Regards,
Siham

**Comment by sophia-wright-blue** [comment]
thank you for replying, @SihamTabik , you mention in your paper "Automatic Handgun Detection Alarm in Videos Using Deep Learning" that for future work, you were preprocessing the videos (increasing their contrast and luminosity), and evaluate dierent CNNs-based classier. Are you considering publishing this future work? thanks,

**Comment by SihamTabik** [comment]
Yes, actually, we've just got the brightness guided model accepted in Neurocompting. I'will send you the link very soon. Best regards

**Comment by sophia-wright-blue** [comment]
thank you for replying @SihamTabik , looking forward to working with your brightness guided model! thanks,
